src/modules/ModuleStaking/Pool.vue

```diff
@@ -135,7 +136,7 @@ wheneverDone(withdrawState, (result) => {
   if (result.fulfilled) {
     const { earned } = result.fulfilled.value
     const formatted = formatCurrency({
-      amount: earned,
+      amount: earned!,
       symbol: { str: props.pool.rewardToken.symbol, position: 'right' },
     })
```

Comment:
Please use `invariant(earned)`, not `!`.

Reply:
Done